client/src/pages/Dashboard/EditProfilePage.tsx

```diff
@@ -11,271 +11,287 @@ import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
 import TextareaAutosize from '@material-ui/core/TextareaAutosize';
 import Typography from '@material-ui/core/Typography';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 import Container from '@material-ui/core/Container';
-import MuiPhoneNumber from "material-ui-phone-number";
+import MuiPhoneNumber from 'material-ui-phone-number';
+import axios from 'axios';
 
 const theme = {
-    spacing: 8,
+  spacing: 8,
 };
- 
-  
-const useStyles = makeStyles((theme) => ({
-paper: {
-    marginTop: theme.spacing(8),
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-},
 
-form: {
-    width: '100%', // Fix IE 11 issue.
-    marginTop: theme.spacing(3),
-},
-submit: {
-    margin: theme.spacing(3, 0, 2),
-    padding: "0.5rem",
-    width: "10%"
-},
-textbox: {
-    width: '100%'
-},
-typography:{
-    fontWeight: "550",
-    fontSize: "18px"
-},
-gridcontainer:{
-    padding: theme.spacing(1)
-},
-heading:{
-    fontWeight: "550"
-}
-}));
- 
- const EditProfile = () => {
+const EditProfile = () => {
+  /* let [ phone, setPhoneNumber ] = useState(''); */
```

Comment:
Please remove

Note: For future use, consider using 'const' instead of 'let' while defining state